src/tiled/commanddialog.cpp

```diff
@@ -71,20 +71,21 @@ void CommandDialog::closeEvent(QCloseEvent *event)
 
 void CommandDialog::setShortcut(const QKeySequence &keySequence)
 {
-    if (!mUi->keySequenceEdit->isEnabled())
-        mUi->keySequenceEdit->setEnabled(true);
-
     const QModelIndex &current = mUi->treeView->currentIndex();
     if (current.row() < mUi->treeView->model()->rowCount(QModelIndex()))
         mUi->treeView->model()->setShortcut(current, keySequence);
 }
 
 void CommandDialog::updateKeySequenceEdit(const QModelIndex &current, const QModelIndex &)
 {
-    if (current.row() < mUi->treeView->model()->rowCount(QModelIndex()) - 1)
+    if (current.row() < mUi->treeView->model()->rowCount(QModelIndex()) - 1) {
+        mUi->keySequenceEdit->setEnabled(true);
         mUi->keySequenceEdit->setKeySequence(mUi->treeView->model()->shortcut(current));
-    else
+    }
+    else {
```

Comment:
Coding style: these should go on the same line `} else {`. I think it'll be my last comment. :-)